src/Microsoft.Extensions.Configuration/ChainedConfigurationProvider.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Extensions.Configuration
+{
+    public class ChainedConfigurationProvider : ConfigurationProvider
+    {
+        private readonly IConfiguration _config;
+
+        public ChainedConfigurationProvider(IConfiguration config)
```

Comment:
Where's the prefix support? https://github.com/aspnet/benchmarks/blob/master/src/Benchmarks/ConfigurationConfigurationProvider.cs#L49